package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "mmt",
+  "version": "1.0.0",
```

Comment:
Is this just for our benefit, or do we need to keep this updated somehow (i.e. the version in Bamboo)?